src/app/projects/add-edit-project/add-edit-project.component.ts

```diff
@@ -527,13 +576,6 @@ export class AddEditProjectComponent implements OnInit, AfterViewInit, OnDestroy
       projectData.centroid = [-123.3656, 48.4284];
     }
 
-		// Set the text background colour for the shape file colour picker.
-    if (projectData.shapeFileColour) {
-      this.shapeFileColour = projectData.shapeFileColour;
-      this.shapeFileTextColour = this.colourIsBright(this.shapeFileColour) ? '#000000' : '#ffffff';
-    }
-
-		// Choose which project phase list is shown: regular phases or forest phases.
     if (projectData.projectTypes) {
       this.projectTypes = projectData.projectTypes || this.projectTypes;
 			this.chosenPhases = projectData.projectTypes?.find((type) => 'Forest Landscape Planning' === type.name)?.checked ? Constants.FOREST_PHASES : Constants.DEFAULT_PHASES;
```

Comment:
just tab again

Reply:
I'm not sure what you mean by this. It looks like I knocked out the comment above by accident. I'll restore it. Is that what you mean?

Reply:
No it's the tab indent:

      this.projectTypes = projectData.projectTypes || this.projectTypes;
		here=>	this.chosenPhases...

Reply:
It's technically not part of your own code so feel free to ignore it.

Reply:
I can fix it, np